plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessStandardExit.sol

```diff
@@ -22,6 +22,11 @@ library PaymentProcessStandardExit {
         uint160 indexed exitId
     );
 
```

Comment:
We should refactor the function so it follows Checks-Effects-Interactions pattern -> https://github.com/omisego/plasma-contracts/blob/eb53c85aa821f53f49a2fc2d7b31aa548a219766/plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessStandardExit.sol#L69